shopinvader_sale_update_price/models/shopinvader_backend.py

```diff
@@ -45,6 +44,9 @@ def launch_sale_price_update(self, domain=False):
         """
         domain = domain or []
         if domain:
+            # Fix domain when integers are received. TODO: Find a better way to do that.
```

Comment:
Which is the case where you have an interger as value in the first item of the domain ?